taxi/forms.py

```diff
@@ -0,0 +1,69 @@
+from django import forms
+from django.core.exceptions import ValidationError
+from django.contrib.auth.forms import UserCreationForm
+from django.contrib.auth import get_user_model
+
+from taxi.models import Driver, Car
+
+
+class DriverLicenseUpdateForm(forms.ModelForm):
+    LICENSE_LEN = 8
+
+    class Meta:
+        model = Driver

```

Comment:
You should use `get_user_model()` instead of `Driver` as per the checklist. This is to ensure that the code will still work if the AUTH_USER_MODEL setting changes.